### PR TITLE
Add Patch VM Scale Set API and vm health status

### PR DESCRIPTION
**Pull request by hyonholee**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json [hunk 1]
```diff
@@ -1368,6 +1368,45 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/instanceView": {
+      "get": {
+        "tags": [
+          "VirtualMachines"
+        ],
+        "operationId": "VirtualMachines_InstanceView",
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json]: Should be InstanceView_Get.

Comment (jhendrixMSFT) [reply specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json]: After looking at other APIs I take that back, leaving it as-is works better.

**Review by jhendrixMSFT — COMMENTED** [review]
I don't see anything too concerning here.  Waiting for feedback from ARM team.

File: specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json [hunk 2]
```diff
@@ -1368,6 +1368,45 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/instanceView": {
```
Comment (ravbhatnagar) [inline specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json]: Instance view is already a property on the virtualMachine. Why do you want to expose it through a separate API? This makes it look like instanceView is a child of virtualMachine which it is not.

**Review by ravbhatnagar — CHANGES_REQUESTED** [review]

**Comment by jhendrixMSFT** [comment]
@hyonholee have you had a chance to review the feedback?

**Comment by salameer** [comment]
@hyonholee, kidnly respond to the feedback to avoid closing this PR for inactivity :)

**Comment by jhendrixMSFT** [comment]
@hyonholee Any update?  Please note that this PR will be closed by the end of the week if no progress is made.

**Comment by jhendrixMSFT** [comment]
Closing due to inactivity.  Please feel free to reopen as required.

**Pull request closed without merging**